docs/user_manual/sequencer_runs.md

```diff
@@ -0,0 +1,153 @@
+
+To remove a container from a run, go the the Edit Run page for the appropriate run, find the Sequencing Containers
+section, and select the container that you'd like to remove. Click the "Remove" button in the toolbar at the top of the
+table to remove the container.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="External Links" %}
+
+If you have a web resource that includes pages for each of your runs, and the URL for those are predictable based on
+the run ID, name, or alias in MISO, MISO can be configured to automatically link to these. For example, links could be
+added to `https://www.example.com/runs/{run-alias}`. These can be set up for individual sequencing platforms, so if you
+have a report that is specific to PacBio runs, there will be no link on Illumina run pages. Talk to your MISO
+administrator about setting these up.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Notes" %}
+
+The Edit Run page includes a Notes section. Notes can be used to record additional run information that is otherwise
+not recorded in MISO. More information on working with notes can be found in the [Notes section](notes.html).
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Attaching Files" %}
+
+You can attach any type and number of files to a run in MISO. This feature might be used to attach a report of run
+performance, or any other run-related files. For more information, see the [Attachments section](attachments.html).
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Related Issues" %}
+
+If your MISO instance is configured for integration with an issue tracker such as JIRA, the Edit Run page will include
+a Related Issues section. This section will list links to tickets in the issue tracker that are related to the run. An
+issue is considered to be related to the run if the run alias appears in the title, text, or comments of the ticket.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Downloading a Sample Sheet" %}
+
+Sample sheets are required for some sequencing analysis software in order to identify and demultiplex the libraries.
+MISO is able to automatically generate these sample sheets for several versions of BCL2FASTQ/CASAVA and Cell Ranger. To
+download these sheets, go to the Edit Run page. Near the top of the screen, there are buttons for downloading the
+sample sheets.
+
+{% include userman-toplink.md %}
+
+
+
+{% assign sub = sub | plus: 1 %}
+{% include userman-heading.md section=page.section sub=sub title="Automatically Populate Runs using RunScanner" %}
```

Comment:
RunScanner -> Run Scanner (and elsewhere in this section, as well as in the table of contents)